### ISSUE TITLE
Remove facial expression from animals

### ISSUE THREAD

**Issue by Crissov** [EmojiTwo/emojitwo#80]
https://github.com/Ranks/emojione-assets/issues/5
https://github.com/Ranks/emojione/issues/339
@hotgeart

> The emojis must be neutral. For example : http://emojipedia.org/bear-face/
>
> Why the bear look mad? Same for tiger, gorilla, camel, ...

![U+1F42D ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f42d.png) ![U+1F42E ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f42e.png) ![U+1F42F ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f42f.png) ![U+1F430 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f430.png) ![U+1F431 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f431.png) ![U+1F432 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f432.png) ![U+1F434 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f434.png) ![U+1F435 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f435.png) ![U+1F436 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f436.png) ![U+1F437 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f437.png) ![U+1F438 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f438.png) ![U+1F439 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f439.png) ![U+1F43A ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f43a.png) ![U+1F43B ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f43b.png) ![U+1F43C ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f43c.png) ![U+1F981 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f981.png) ![U+1F984 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f984.png) ![U+1F985 ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f985.png) ![U+1F98A ](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f98a.png) ![U+1F417 🐗](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f417.png) ![U+1F42A 🐪](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f42a.png) ![U+1F98C 🦌](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f98c.png) ![U+1F98D 🦍](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f98d.png) ![U+1F98F 🦏](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f98f.png)

**Comment by KAMiKAZOW** [comment]
Are you aware of an existing effort for sequences to combine animals with faces to give them emotional expressions? E.g. bear + ![U+1f620](https://rawgit.com/EmojiTwo/emojitwo/master/png/1f620.png) = angry bear

**Comment by Crissov** [comment]
No, I would track it at https://github.com/Crissov/unicode-proposals/issues/278
